variables.tf

```
@@ -8,6 +8,12 @@ variable "cidr_block" {
   description = "The CIDR block for the VPC"
 }
 
+variable "enable_nat_gateway" {
+  type        = bool
+  default     = true
+  description = "Should be true to provision NAT Gateways for each private subnet"
```

Hi @lassebenni! Very minor suggestion, we prefer to say "Set to ...":

```suggestion
  description = "Set to true to provision a NAT Gateway for each private subnet"
```

---

@shoekstra Definitely a good practice, thanks!